osrsbox/items_api_examples/guess_the_item_game.py

```diff
@@ -0,0 +1,29 @@
+"""
+Description:
+A very simple 'game' that gets a random item's examine text from the osrsbox-db item database
+and asks the user to guess the name of the item
+"""
+
+
+from osrsbox import items_api
+import random
+
+if __name__ == "__main__":
+    all_db_items = items_api.load()
+    item_found = False
+    item_name = ""
+    while item_found == False:
+        random_id = random.randint(1,len(all_db_items))
+        for item in all_db_items:
+            # Discard items with examine text on None, this is not a required field.
+            if item.examine != None and item.id == random_id:
```

Comment:
Flake8 is complaining about this one too. 

`if item.examine != None`

Should be:

`if item.examine is not None`